src/style-spec/function/index.js

```diff
@@ -90,6 +90,9 @@ function createFunction(parameters: FunctionParameters, propertySpec: StylePrope
                 throw new Error(f.zoomCurve ? f.zoomCurve.error : 'Invalid zoom expression');
             }
         }
+
+        // useful for debugging, especially for converted stop functions
+        f.rawExpression = expr;
         return f;
     } else {
         console.log(JSON.stringify(expr, null, 2));
```

Comment:
same as above perhaps 💭

Reply:
Oops, yeah this is leftover, will remove